tools/rts_verifier/operation.py

```diff
@@ -64,7 +64,7 @@ def __repr__(self):
         return f"{self.__dict__}"
 
     def __extract_file(self, line):
-        matches = re.search('file:([\w_\.]+)', line)
+        matches = re.search('file:([\w_\.]+)', line) or re.search('tiered:([\w_\.]+)', line)
```

Comment:
@korteland Can you please review this part of the code ?

Reply:
This would be a little simpler as `matches = re.search('(?:(file|tiered)):([\w_\.]+)', line)` - note that it's a non-capturing group at the start.